keanu-project/src/test/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensorTest.java

```diff
@@ -239,6 +239,15 @@ public void canApplySqrt() {
 
     @Test
     public void canBroadcastMultiplyRank4ContainingVectorAndMatrix() {
+
+        /*
+          a = np.array([1, 2, 3, 4, 5, 6, 7, 8]).reshape(2,2,2,1)
+          b = np.array([1, 2, 3, 4]).reshape(2,2,1,1)
+          ab = a * b
+          print(ab)
+          print(np.shape(ab))
+         */
+
```

Comment:
Awesome idea.  Is there a comment anywhere making it clear if people aren't goint to immediately recognise this as NumPy stuff?

Reply:
I meant to have these changes go in on a different PR where it was better explained. @GeorgeNash is looking at further Numpy checking. I'll add a little line on each of them saying it's numpy.

Reply:
👍